Cauldron/Assets/Scripts/GameModes/PrestigeGameMode.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using CauldronCodebase.GameStates;
+using UnityEngine;
+using Zenject;
+
+namespace CauldronCodebase
+{
+    public class PrestigeGameMode : GameModeBase
+    {
+        [SerializeField] private int targetDays = 25;
+        protected string achievIdents;
+        
+        protected GameStateMachine gameStates;
+        protected GameDataHandler gameData;
+        private IAchievementManager achievement;
+
+        [Inject]
+        public void Construct(GameStateMachine gameStates, GameDataHandler gameData, IAchievementManager achievement)
```

Comment:
We don't need to call this method Construct to make it the target for injects, the attribute does the job. I suppose we can name it ConstructInternal and avoid Construct hierarchy))